packages/slice-machine/src/hooks/usePromptToCreateContentExperiment.ts

```diff
@@ -4,5 +4,7 @@ type UsePromptToCreateContentExperimentReturnType = { eligible: boolean };
 
 export function usePromptToCreateContentExperiment(): UsePromptToCreateContentExperimentReturnType {
   const variant = useExperimentVariant("slicemachine-prompt-to-create-content");
-  return { eligible: variant?.value === "on" };
+
+  // serve the experimental version to user with Telemetry deactivated
```

Comment:
If we do this, then we won't be able to understand how well it performs, because part of the experiment will come from a different, non controlled scenario, right?

Reply:
I'm kind of not sure how to tackle this one. 
For me control version of ab-test is the old version, and the experiment is the new version.
But here product decision was to serve new version by default, also to users that are not in the ab-test at all (the case of disabled telemetry). That's why I added this fallback.
But would that influence the experiment metrics? We don't track actions of users with telemetry disabled, right? Or am I missing something here?

Reply:
i guess you're right (if we assume that no variant means that segment events are not sent either)

but in that case, i would have modeled this the other way around: i would create a flag to opt in to the version that is not the default one

Reply:
I thought about that, but I'm not 100% sure how Amplitude will treat that (as in their docs the control version is the old version)